website/source/docs/builders/vagrant.html.md

```diff
@@ -72,7 +72,7 @@ one, by using `global_id` instead of `source_box`.
 
 -   `vagrantfile_template` (string) - a path to a golang template for a
     vagrantfile. Our default template can be found
-    [here](https://github.com/hashicorp/packer/tree/master/builder/vagrant/step_initialize_vagrant.go#L23-L30). So far the only template variables available to you are {{ .BoxName }} and
+    [here](https://github.com/hashicorp/packer/tree/master/builder/vagrant/step_create_vagrantfile.go#L23-L30). So far the only template variables available to you are {{ .BoxName }} and
```

Comment:
Nice one !
Here, maybe the permalink will make more sense so that we don't have to update this file all the time the file changes, what do you think ?
```suggestion
    [here](https://github.com/hashicorp/packer/blob/a6d1d852bb0643e3593cc5d3dfe5ed80c4409b65/builder/vagrant/step_create_vagrantfile.go#L23-L30). So far the only template variables available to you are {{ .BoxName }} and
```